src/components/MinecraftCommands.ts

```diff
@@ -0,0 +1,82 @@
+type CommandData = [string, string[], number | string];
```

Comment:
this should rather be an interface with properly named properties, looks a little confusing without description